contracts/upgradeable_contracts/Validatable.sol

```diff
@@ -0,0 +1,24 @@
+pragma solidity 0.4.23;
+import "../IBridgeValidators.sol";
+import "../upgradeability/EternalStorage.sol";
+import "./Ownable.sol";
+import "../upgradeability/OwnedUpgradeabilityProxy.sol";
+
+
+contract Validatable is EternalStorage, Ownable, OwnedUpgradeabilityProxy {
+
+    function validatorContract() public view returns(IBridgeValidators) {
+        return IBridgeValidators(addressStorage[keccak256("validatorContract")]);
+    }
+
+    modifier onlyValidator() {
+        require(validatorContract().isValidator(msg.sender));
+        _;
+    }
+
+    function upgradeSecurityRoles(address _newOwner) public {
```

Comment:
It is not safe to do since anyone who run this method after the contract upgrade first could set up the owner and capture control on the bridge contract. Consider to add the modifier `onlyProxyOwner`.

Reply:
Thanks. Fixed

Reply:
Should this method be included in git at all? It is needed just for upgrade in case of already deployed contracts (MakerDAO-xDAI).

Reply:
The intention was to provide a way to upgrade already deployed contracts. It makes sense to not include it in git. I'll remove it and add it only on gist files

Reply:
Done